bokeh/models/math_text.py

```diff
@@ -0,0 +1,70 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2012 - 2021, Anaconda, Inc., and Bokeh Contributors.
+# All rights reserved.
+#
+# The full license is in the file LICENSE.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+''' Display a mathematics notation from a string value.
+'''
+
+#-----------------------------------------------------------------------------
+# Boilerplate
+#-----------------------------------------------------------------------------
+from __future__ import annotations
+
+import logging # isort:skip
+log = logging.getLogger(__name__)
+
+#-----------------------------------------------------------------------------
+# Imports
+#-----------------------------------------------------------------------------
+
+# Bokeh imports
+from ..core.properties import Nullable, String
+from ..model import Model
+
+#-----------------------------------------------------------------------------
+# Globals and constants
+#-----------------------------------------------------------------------------
+
+__all__ = (
+    'MathText',
+)
+
+#-----------------------------------------------------------------------------
+# General API
+#-----------------------------------------------------------------------------
+
+class MathText(Model):
+    """
+    Render mathematical content using using [LaTeX](https://en.wikipedia.org/wiki/LaTeX) notation
+
+    .. note::
+        Bokeh uses [MathJax](https://www.mathjax.org) to render text containing mathematical notation.
+
+        MathJax only supports math-mode macros (no text-mode macros). You
+        can see more about differences between standard TeX/LaTeX and MathJax
+        here: https://docs.mathjax.org/en/latest/input/tex/differences.html
+    """
+
+    text = Nullable(String, help="""
```

Comment:
Why is this nullable if the (bokehjs) implementation defines this property as a required string? This either needs to be `String` if `""` default value is desired` or `NonNullable(String)` otherise.